plugin.video.openingsmoe/main.py

```diff
@@ -0,0 +1,22 @@
+import sys
+from urllib import urlencode
+from urlparse import parse_qsl
+import xbmcgui
```

Comment:
please remove unused imports. You don't use `xbmcgui` and `xbmcplugin`. The fact you don't use `xbmcplugin` is a pretty good indicator your addon is not a plugin but a script.